db.go

```diff
@@ -345,45 +317,26 @@ func (db *DB) GetAuthenticatedUser(sessionID string) (AuthenticatedUser, error)
 	return v, nil
 }
 
-func (db *DB) Signer() core.Signer {
-	var v RSAKey
-	db.f.Read(func(db *schema) {
-		v = db.OIDCSigningKey
-	})
-	if v.KeyID == "" {
-		panic("signing key not set")
-	}
-	return &oidcSigner{key: v}
-}
-
-func (db *DB) KeySource() discovery.KeySource {
-	var v RSAKey
-	db.f.Read(func(db *schema) {
-		v = db.OIDCSigningKey
-	})
-	if v.KeyID == "" {
-		panic("signing key not set")
-	}
-	return &staticKeySource{key: v}
-}
-
 func (db *DB) SessionManager() core.SessionManager {
 	return &sessionManager{f: db.f}
 }
 
-const rsaKeyBits = 2048
+func (db *DB) GetKeyset(ks Keyset) DBKeyset {
```

Comment:
nit but I would change this to just take a `name string` (ditto for `PutKeySet`)